### ISSUE TITLE
The ~INT_BAYESIAN~ flag does not seem to work

### ISSUE THREAD

**Issue by cerebrai** [KevinMusgrave/powerful-benchmarker#89]
Hi,

I have a custom loss function that has a hyper-parameter that only takes integer values. I used the following command 

--loss_funcs\~OVERRIDE\~ {metric_loss: {MyCustomLoss: {t_k\~INT_BAYESIAN\~: [0, 10]}}},

however, the first bayesian iteration sets a float value of around 8.3 for t_k. What am I missing here?

Note: As a workaround, I have type-casted t_k to be int in my code but I am afraid it might affect the optimization process.

**Comment by KevinMusgrave** [comment]
I'm not sure why that is happening. However, I think your workaround might be equivalent, according to this issue: https://github.com/facebook/Ax/issues/133
